cmake/README.md

```diff
@@ -1492,6 +1492,11 @@ target API.
   </dl>
   </td>
 </tr>
+<tr>
+  <td><code>BIN2C</code> (CUDA only)</td>
+  <td>Path to bin2c executable</td>
+  <td>1st in path (if found)</td>
```

Comment:
@rbberger @akohlmey I am not super happy with the text for `value`. Any suggestions.

Reply:
`First match in $PATH` ?

Reply:
I was also not sure if this should go into the `value` column or under description.